WalletWasabi/WabiSabi/Backend/WabiSabiConfig.cs

```diff
@@ -205,6 +205,14 @@ public WabiSabiConfig(string filePath) : base(filePath)
 	[JsonProperty(PropertyName = "AllowP2trOutputs", DefaultValueHandling = DefaultValueHandling.Populate)]
 	public bool AllowP2trOutputs { get; set; } = true;
 
+	[DefaultValue(false)]
+	[JsonProperty(PropertyName = "AllowP2pkhOutputs", DefaultValueHandling = DefaultValueHandling.Populate)]
+	public bool AllowP2pkhOutputs { get; set; } = false;
+
+	[DefaultValue(false)]
+	[JsonProperty(PropertyName = "AllowP2shOutputs", DefaultValueHandling = DefaultValueHandling.Populate)]
+	public bool AllowP2shOutputs { get; set; } = false;
```

Comment:
Why false by default? Is there some cases where it would be annoying for a coordinator to have this? I can only see upsides.

Reply:
Yes, you're right. Next PR will switch them to true.